tests/cxx/unit_tests/chemical_system/molecule/molecule.cpp

```diff
@@ -189,6 +199,25 @@ TEST_CASE("Molecule Class") {
         REQUIRE(hd.multiplicity() == 3);
     }
 
+    SECTION("multiplicity_data()") {
+        REQUIRE(defaulted.multiplicity_data() == nullptr);
+        REQUIRE(*mol.multiplicity_data() == 2);
+    }
+
+    SECTION("charge_data() const") {
+        REQUIRE(defaulted.charge_data() == nullptr);
+        REQUIRE(*mol.charge_data() == 0);
+    }
```

Comment:
I think this is supposed to be for `multiplicity_data() const`. Also, as with the above `charge_data` case, I think these need to be `std::as_const`, right?

Reply:
You're right on both of these changes. Too much copy/pasting...